DEVELOPER_GUIDE.md

```diff
@@ -1,32 +1,33 @@
-# WSO2 API Manager extension with Ping Intelligence
+# WSO2 API Manager extension with PingIntelligence
 
 ## What is PingIntelligence for APIs?
-PingIntelligence for APIs uses artificial intelligence (AI) to expose active APIs, identify and automatically block cyber attacks on APIs and provide detailed reporting on all API activity. Deployed on premises, in public clouds or in hybrid clouds, the solution monitors API traffic across the environment. It uses AI and machine learning models to detect anomalous API behavior without relying on specifically defined policies or prior knowledge of attack patterns in which, can stop new and constantly changing attacks. Using continuous learning capabilities, it becomes more accurate at identifying and blocking attacks over time. 
+PingIntelligence for APIs uses artificial intelligence (AI) to expose active APIs, identify and automatically block cyber attacks on APIs and provide detailed reporting on all API activity. You can deploy the PingIntelligence solution on premises, in public clouds, or in hybrid clouds to monitor API traffic across the environment. PingIntelligence uses AI and machine learning models to detect anomalous API behavior, without relying on specifically defined policies or prior knowledge of attack patterns, in order to stop new and constantly changing attacks. In addition, PingIntelligence uses its continuous learning capabilities to become more accurate at identifying and blocking attacks over time.  
 
 
-#### PingIntelligence protects against three main types of attacks, specifically:
+#### Types of Attacks PingIntelligence Protects Against
```

Comment:
Since you are changing the heading, you may have to change the link in README.md.

Reply:
Fixed